multimedia/winff/winff.SlackBuild

```diff
@@ -73,9 +72,8 @@ rm -rf $PKG
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
 rm -rf $PRGNAM-$VERSION
-rm -rf $PRGNAM-$LONGNAM
-tar xvf $CWD/$PRGNAM-$LONGNAM.tar.gz
-cd $PRGNAM-$LONGNAM
+tar xvf $CWD/$PRGNAM-$VERSION.tar.gz
+cd $PRGNAM-$PRGNAM-$VERSION
```

Comment:
You can see the issue here. For github downloads, if you have a different filename to the source directory the basename of the file used in the tar line, then the URL used is likely to give different results with wget and a browser etc.

When I follow the link (here in a browser) to the source from the info file in the PR, it results in the name you use here in `cd` namely, winff-winff-1.6.3.tar.gz. That's because the name of the tag is `winff-1.6.3` and then the project name is prefixed:

![winff](https://github.com/SlackBuildsOrg/slackbuilds/assets/346774/913a65b8-b9d4-4c37-83d9-1d60764fed34)

The fix is to use this full name for all three things: cd, tar, and rm.

Then url in the info file should be:

https://github.com/WinFF/winff/archive/winff-1.6.3/winff-winff-1.6.3.tar.gz

That will give the same filename with wget, a browser, wget with `--content-disposition` etc.

Dave wrote a document with more details here: https://slackbuilds.org/GITHUB_URLs.txt